### PR TITLE
docs(docs-operator): update the readme and guides files

### PR DESCRIPTION
**Pull request by kiragoo**

### CODE REVIEW

File: docs/user-guides/getting-started.md [hunk 1]
```diff
@@ -13,34 +13,34 @@ This project can be run inside a kubernetes cluster or outside of it, by taking
...
+   emqxenterprises.apps.emqx.io                2021-12-01T03:15:58Z
    ```
 
-
 3. Enable RBAC rules for EMQ X Operator pods
 
    ```shell
-   $ kubectl create -f https://raw.githubusercontent.com/emqx/emqx-operator/main/config/samples/operator/namespace.yaml
-   $ kubectl create -f https://raw.githubusercontent.com/emqx/emqx-operator/main/config/samples/operator/rbac.yaml
+   kubectl create -f https://raw.githubusercontent.com/emqx/emqx-operator/main/config/samples/operator/namespace.yaml
```
Comment (Rory-Z) [inline docs/user-guides/getting-started.md:36]: Delete this line， this file is not exists

File: docs/user-guides/getting-started.md [hunk 2]
```diff
@@ -66,16 +89,16 @@ This project can be run inside a kubernetes cluster or outside of it, by taking
 3. Create RBAC objects from manifest file
 
    ```shell
-   $ kubectl create -f https://raw.githubusercontent.com/emqx/emqx-operator/main/config/samples/operator/operator_namespace.yaml
-   $ kubectl create -f https://raw.githubusercontent.com/emqx/emqx-operator/main/config/samples/operator/rbac.yaml
+   kubectl create -f https://raw.githubusercontent.com/emqx/emqx-operator/main/config/samples/operator/operator_namespace.yaml
```
Comment (Rory-Z) [inline docs/user-guides/getting-started.md:92]: Delete this line， this file is not exists

File: docs/user-guides/getting-started.md [hunk 3]
```diff
@@ -128,6 +191,8 @@ This project can be run inside a kubernetes cluster or outside of it, by taking
    $ kubectl get pods
    NAME              READY   STATUS    RESTARTS   AGE
    emqx-0   1/1     Running   0          22m
+   emqx-1   1/1     Running   0          22m
+   emqx-3   1/1     Running   0          22m
```
Comment (Rory-Z) [inline docs/user-guides/getting-started.md:195]:
```suggestion
   emqx-2   1/1     Running   0          22m
```

File: README.md [hunk 4]
```diff
@@ -21,9 +21,9 @@ The EMQ X Operator requires a Kubernetes cluster of version `>=1.20.0`.If you ar
 ## CustomResourceDefinitions
 
 A core feature of the EMQ X Operator is to monitor the Kubernetes API server for changes to specific objects and ensure that the running EMQ X deployments match these objects.
-The Operator acts on the following [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/):
+The Operator acts on the following [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/).
 
-* **`Cluster`**, which defines a desired EMQ X Broker Cluster deployment.
+The examples of EMQ X Broker see the [emqx.yaml](config/samples/emqx/emqx.yaml) and [emqx-ee.yaml](config/samples/emqx/emqx-ee.yaml).
```
Comment (Rory-Z) [inline README.md:26]:
```suggestion
The examples of EMQ X Broker see [emqx.yaml](config/samples/emqx/emqx.yaml) and the examples of EMQ X Enterprise see[emqx-ee.yaml](config/samples/emqx/emqx-ee.yaml).
```

Comment (kiragoo) [reply README.md:26]: tks

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**